### ISSUE TITLE
Autocorrect/suggestions

### ISSUE THREAD

**Issue by FlavioMala** [ubports/morph-browser#406]
Note: I read something on the subject, but I can't find it anymore...

Device: BQ E5
Ch. : dev

A few months ago the autocorrect/suggestions in the browser disappeared (after an upgrade/update). I can't activate them anymore despite several attempts (including language change, now in Spanish).

Everything "on" in "settings":
![screenshot20201114_124035325](https://user-images.githubusercontent.com/50835740/99146437-783b1100-2678-11eb-991d-79cb4a0cd33d.png)
...autocorrect/suggestions in SMS:
![screenshot20201114_124218980](https://user-images.githubusercontent.com/50835740/99146466-be907000-2678-11eb-8705-277c13a60055.png)
...missing autocorrect/suggestions in web browser:
![screenshot20201114_124027211](https://user-images.githubusercontent.com/50835740/99146488-f3042c00-2678-11eb-9d75-90156fcaac64.png)

**Comment by Fuseteam** [comment]
so far no obvious logs in `.cache/upstart/maliit-server.log`

**Comment by iam01110011** [comment]
I still having issues re disappearing auto correct on apps such as morph and axolotl post phone updates. any conclusions reached out there in the wilds of the interwebs? : D

**Comment by Fuseteam** [comment]
the most i've found so far is that it is related to the webengine upgrade

**Comment by FlavioMala** [comment]
.cache/upstart/maliit-server.log
[maliit.txt](https://github.com/ubports/morph-browser/files/5769662/maliit.txt)

**Comment by Fuseteam** [comment]
@FlavioMala those warnings are there regardless whether it works or not, the issue is related to the newer webengine breaking it somehow

**Comment by FlavioMala** [comment]
Yes, if there is/are other thing/s, file/s, etc. useful "from my phone"... let me know it!

**Comment by dano6** [comment]
Maybe related with https://bugreports.qt.io/browse/QTBUG-71995

**Comment by balcy** [comment]
very good hint, thanks !
I've applied the patch of that bug report in https://gitlab.com/ubports/core/packaging/qtwebengine/-/merge_requests/3 which hopefully brings back the predictive text.

**Comment by FlavioMala** [comment]
Is it already implemented in any updates? At the moment I don't see any differences...
(E5 ch dev "674" March 7 2021)

**Comment by dano6** [comment]
It wasn't merged yet. You can test it with `ubports-qa install PR_qtwebengine_3`

**Comment by FlavioMala** [comment]
Wow ! Thank you very much !!!
![screenshot20210310_193058550](https://user-images.githubusercontent.com/50835740/110679613-f26b4480-81d7-11eb-9a62-f071c20de0d5.png)

**Comment by iam01110011** [comment]
Omg!!!! Amazing yeah!!!! It worked for me too! Thanks!!!On 10 Mar 2021 6:37 pm, FlavioMala ***@***.***> wrote:
Wow ! Thank you very much !!!

**Comment by FlavioMala** [comment]
Aaaccssss!
No more suggestions after update to (devel) "694" (today).
Also "sudo ubports-qa install PR_qtwebengine_3" doesn't work.

Until "687" (27/03/2021) was fine (and without "sudo ubports-qa install PR_qtwebengine_3").

**Comment by Fuseteam** [comment]
i believe ubports-qa is currently broken, i am not sure the pr in question was ever merged
